pallets/schemas/src/benchmarking.rs

```diff
@@ -0,0 +1,35 @@
+use frame_benchmarking::{benchmarks, whitelisted_caller};
+use frame_support::{assert_ok, ensure};
+use frame_system::RawOrigin;
+use sp_std::vec::Vec;
+
+use crate::Pallet as SchemasPallet;
+
+use super::*;
+
+fn random_schema() -> Vec<u8> {
+	Vec::from("aasdf,sfdfb,eeec,dfsdfs,egsdkld,sldfklzz,sffs,sdjkl,lslsls,lfkjsf".as_bytes())
+}
+
+fn register_random_schema<T: Config>(sender: T::AccountId) -> DispatchResult {
+	SchemasPallet::<T>::register_schema(RawOrigin::Signed(sender).into(), random_schema())
+}
+
+benchmarks! {
+	register_schema {
```

Comment:
I think schema registration benchmark should be dependent on the schema size as a param. You can take a look at benchmark for `add` in messages pallet as an example.